tasks/lib/index.js

```diff
@@ -121,10 +124,8 @@ var createAndUploadArtifacts = function (options, done) {
 
     artifactStream.on('data', function(chunk) {
 
-        var binaryChunk = chunk.toString('binary');

```

Comment:
I'm not so sure about removing this line.  It seems to be working for others (I'm assuming by lack of other bugs).  I know the unit tests have run successfully.  Would you be willing to test out on a linux machine as well to ensure we don't introduce a different issue?

Thank you